package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
     "hawk": "3.1.0",
+    "halfpenny": "*",
```

Comment:
i think that someone should be able to download nodeapi, run `npm i`, then run `grunt test`.  in that regard, I think halfpenny should remain a `devDependency`.  To _not_ include it is to expect that you have already downloaded halfpenny in some expected location relative to nodeapi.  not including it i think degrades the OpenSource hackability (pull-hack-PR cycle).

my vote is to keep it in there.